### PR TITLE
Hotfix - #37 - catch also php 7 throwables

### PR DESCRIPTION
**Pull request by Ocramius**
Provides tests for #37

### CODE REVIEW

**Comment by nesl247** [comment]
Would this not fail on PHP 5.6 because the class `Throwable` doesn't exist?

**Comment by msheakoski** [comment]
@nesl247 PHP < 7 doesn't appear to complain when Throwable is inside of a catch block. See example: http://sandbox.onlinephpfunctions.com/code/66997de5290d58dca15462e867224c9da62e3e14

**Comment by nesl247** [comment]
Very interesting. I really hope this gets merged in soon then.

**Comment by Ocramius** [comment]
@nesl247 type-hints are not causing any autoloading, so it would just be ignored ;-)

File: test/DispatchTest.php [hunk 1]
```diff
@@ -199,4 +209,47 @@ public function testShouldAllowDispatchingPsr7Instances()
         $result = $dispatch($route, $err, $request->reveal(), $response->reveal(), $next);
         $this->assertSame($response->reveal(), $result);
     }
+
+    /**
+     * @requires PHP 7.0
```
Comment (weierophinney) [inline test/DispatchTest.php]: That's nifty; didn't know about that annotation!

**Pull request merged**